classes/game.js

```diff
@@ -5,30 +5,54 @@ class Game{
         this.turn = data.turn;
         this.board = new Board(data.board);
         this.me = new Snake(data.you);
+				this.test = data;
     }
     //todo:
     findFood(){
-			let foodx = request.body.board.food[0].x
-			let foody = request.body.board.food[0].y
-			
-			let locx = request.body.you.body[0].x
-			let locy = request.body.you.body[0].y
-			
-			if(locx < foodx){
-				data.move = 'right';
-			}else if(locx > foodx){
-				data.move = 'left';
-			}else if(locy < foody){
-				data.move = 'down';
-			}else if(locy > foody){
-				data.move = 'up';
+			let foodLength = Object.keys(this.test.board.food).length;
+			let foodArr = this.test.board.food;
+			
+			let closex = 0;
+			let closey = 0;
+			let foodDistanceSquare = 99999999999999;
+			
+			let locx = this.test.you.body[0].x
+			let locy = this.test.you.body[0].y
+			
+			for (let i = 0; i < foodLength; i++) {
```

Comment:
Doesn't really matter, but this could just be a regular forEach loop, then you don't need to even do the foodLength thing at all

```
foodArr.forEach(function(food){
    let diffx = food.x - locx;
    ...
});
```